start_gomoku.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 DEFAULT_DYLIB_PATH=target/release/librust_ext.dylib
+DEFAULT_PYTHON_BIN=bin/python3
```

Comment:
still not working for me with the venv folder, I changed DEFAULT_PYTHON_BIN=bin/python3 to DEFAULT_PYTHON_BIN=venv/bin/python3, but I think the issue actually comes from my side so I guess you can let it like this its not a big deal ^^, It should work properly it there isn't any venv folder (with bin/python link inside) that has already been created locally